.gitignore

```diff
@@ -1,17 +1,9 @@
-*.pyc
-*.pyo
-*.gourmet-plugin
-*.desktop
-*~
-*#*
-.project
-.pydevproject
-.settings/
+# Byte-compiled / optimized / DLL files
+__pycache__/
+
+
+# Distribution / packaging
 *.egg-info/
-debian
-dist/
```

Comment:
This contains the output of `python setup.py sdist` or `bdist_wheel`.  
Since it's generated content, which we let the CI build anyway, they oughtn't be committed.